src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -387,6 +875,7 @@ public void UnsupportedCallingConventionInMetadata()
 .class public auto ansi beforefieldinit C
        extends [mscorlib]System.Object
 {
+    .field public method vararg void*() 'Field'
```

Comment:
>method vararg void*() [](start = 18, length = 21)

It would be good to assert value of MethodSymbol.IsVararg for this function pointer signature.